setup/flex.rst

```diff
@@ -111,7 +111,7 @@ Symfony application by executing the following command:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/skeleton my-project
+    $ composer create-project symfony/skeleton my-project 3.4
```

Comment:
IMO we should use a constraint that will pick the latest patch release for 3.4 like this: `composer create-project symfony/skeleton:^3.4 my-project`